e2e/cypress/integration/notifications/desktop_notifications_spec.js

```diff
@@ -418,6 +409,156 @@ describe('Desktop notifications', () => {
             });
         });
     });
+
+    it('MM-T491 - Channel notifications: No desktop notification when in focus', () => {
+        cy.apiCreateUser().then(({user}) => {
+            cy.apiAddUserToTeam(testTeam.id, user.id);
+            cy.apiLogin(user);
+
+            cy.apiGetChannelByName(testTeam.name, 'Off-Topic').then(({channel}) => {
+                const message = '/echo test 3';
+
+                // # Go to Off topic
+                cy.visit(`/${testTeam.name}/channels/${channel.name}`);
+                spyNotificationAs('withNotification', 'granted');
+
+                // Have another user send a post with delay
```

Comment:
Please add `#` for step or `*` for verification before the comments to help non-technical reader/reviewer be able to follow and compare to test case without trying to understand the code itself.
```suggestion
                // # Have another user send a post with delay
```